servers/pollux/default.nix

```diff
@@ -0,0 +1,163 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+let
+  vs = config.vault-secrets.secrets;
+
+  suitecrm = import ./suitecrm.nix { inherit pkgs; system = pkgs.system; };
+in {
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+    inputs.hermetic.nixosModules.hermetic
+  ];
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+
+  users.users.suitecrm = {
+    isNormalUser = true;
+    home = "/var/www/suitecrm";
+    uid = 1024;
+    group = "suitecrm";
+  };
+
+  users.groups.suitecrm = {
+    gid = 1024;
+  };
+
+  systemd.services.nginx.preStart = pkgs.lib.mkForce "";
```

Comment:
Are you trying to avoid `nginx -t` here? That's not something we should do unless we have do, and it should definitely be documented.

Reply:
yeah was supposed to be a comment saying that its a temp change as Nginx didnt start without it, but the comment got lost somewhere

Reply:
Well what is the error it produced? If it fails testing there's probably a good reason and something to fix

Reply:
None it just freezes, I'm  not sure but I think it tried to contact Vault or something